libs/widget-lib/src/types.ts

```diff
@@ -93,7 +93,7 @@ export interface PartnerFee {
   /**
    * The Ethereum address of the partner to receive the fee.
    */
-  recipient: string
+  recipient: string | Record<SupportedChainId, string>
```

Comment:
I might be confusing with something else, but in the second format, all chains are required, no?

Would this allow to provide only a subset of chains:

```suggestion
  recipient: string | Partial<Record<SupportedChainId, string>>
```

Reply:
Yes, all chains are required, this is mandatory.
I think it's better when users are protected from accidental errors (when they are not specified a recipient for some network)

Reply:
But even for test networks?

Reply:
Yes, it's better to specify the same address as in Mainnet than nothing